Header header

float64 stop_distance_err
